templates/header/topbar_begin.html

```diff
@@ -28,27 +28,29 @@
                 </a>
 
                 <ul class="pure-menu-list pure-menu-right">
-                    <li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover pure-menu-opt-children">
-                        <a href="/about" class="pure-menu-link">
+                    <li class="pure-menu-item pure-menu-has-children pure-menu-opt-children">
+                        <a href="#" class="pure-menu-link">
```

Comment:
Why keeping this element then?

Reply:
When I removed the `<a>` tag and just had the text "About", clicks on "About" didn't do anything.